Lesson8/main.js

```diff
@@ -0,0 +1,20 @@
+const addLoading = () =>{
+  const loading = document.getElementById("js-loading");
+  const loadingImg = document.createElement('img');
+  loadingImg.src = 'loading.gif';
+  loadingImg.id = "loading-img"
+  loading.appendChild(loadingImg)
+}
+
+const promiseReject = new Promise((resolve, reject) => {
+  addLoading();
+  setTimeout(() => {
+    reject("The promise was rejected!");
+  }, 3000);
+});
+
+promiseReject.then((error)　=> {
+})
+.catch((error) =>{
+  console.log(error)
```

Comment:
nits
If you want to output errors to the console, it is better to use the API ``console.error``
[ref](https://www.nxworld.net/js-console-api.html#consoleerror)

```suggestion
  console.error(error)
```

Reply:
Thank you for your sugggestion! 
I did not know that the way to output error message in console.
I changed it to .error()
Please check my latest commit  b4df1a91845e32070e5138a90d080cd2f28b5993